api/registry/models.py

```diff
@@ -192,3 +190,21 @@ class Meta:
                 name="gtc_staking_index_by_staker",
             ),
         ]
+
+
+class AddressList(models.Model):
+    name = models.CharField(max_length=100, db_index=True, unique=True)
+
+
+class AddressListMember(models.Model):
+    address = EthAddressField(null=False, blank=False, max_length=100, db_index=True)
+    list = models.ForeignKey(
+        AddressList,
+        related_name="addresses",
+        on_delete=models.CASCADE,
+        null=False,
+        db_index=True,
+    )
+
+    class Meta:
+        unique_together = ["address", "list"]
```

Comment:
- I would move these 2 models to the `account` app. In my view this is part of the scorer setup (just like weights and customisation). Also in the future we might add customisation related to the allowlist stamp to the `AddressList` model (text, logo, etc ...).
- Also please add a link to the `Community` in `AddressList`, as this list will be owned by a community
- Add `status` flag to the `AddressList` with values `draft(default) | live`, so that we are able to make a given list live and hide it when not required any more.